backend/globaleaks/models/__init__.py

```diff
@@ -952,13 +1081,16 @@ class _User(Model):
     can_postpone_expiration = Column(Boolean, default=True, nullable=False)
     can_grant_access_to_reports = Column(Boolean, default=False, nullable=False)
     can_transfer_access_to_reports = Column(Boolean, default=False, nullable=False)
+    can_download_infected = Column(Boolean, default=False, nullable=False)
     can_redact_information = Column(Boolean, default=False, nullable=False)
     can_mask_information = Column(Boolean, default=True, nullable=False)
     can_reopen_reports = Column(Boolean, default=True, nullable=False)
     can_edit_general_settings = Column(Boolean, default=False, nullable=False)
     readonly = Column(Boolean, default=False, nullable=False)
     two_factor_secret = Column(UnicodeText(32), default='', nullable=False)
     reminder_date = Column(DateTime, default=datetime_null, nullable=False)
+    status = Column(Enum(EnumUserStatus), default='active', nullable=False)
+    fiscal_code = Column(UnicodeText(18), default='', nullable=False)
```

Comment:
TODO: to be renamed  idp_id that is the abstraction we could use at international level as inidicated in https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4107